pkg/cmd/templates/templater.go

```diff
@@ -33,10 +34,25 @@ func ActsAsRootCommand(cmd *cobra.Command, filters []string, groups ...CommandGr
 	return templater
 }
 
+func mainUsageTemplate() string {
```

Comment:
It's similar but the order is slightly different - note how `kubectl` has the usage at the bottom and we don't. Not sure if we want to converge at some point, but currently we don't.